recipe/meta.yaml

```diff
@@ -67,95 +70,104 @@ outputs:
         - cmake -S . -B ./build -G=Ninja && cmake --build ./build -v
     about:
       home: https://developer.nvidia.com/cuda-toolkit
-      license_file: LICENSE
+      license_file:
+        - {{ PREFIX }}/targets/{{ ctk_targets[target_platform] }}/LICENSE
       license: LicenseRef-NVIDIA-End-User-License-Agreement
       license_url: https://docs.nvidia.com/cuda/eula/index.html
       summary: Compiler for CUDA applications.
       description: |
         Compiler for CUDA applications.
       doc_url: https://docs.nvidia.com/cuda/index.html
+  {% endif %}
 
-  - name: cuda-nvcc_{{ target_platform }}
+  - name: cuda-nvcc_{{ cross_target_platform }}
     build:
-      noarch: generic
-    run_exports:
-      strong:
-        - cuda-version >={{ cuda_version }},<{{ cuda_version.split(".")[0]|int + 1 }}
+      run_exports:
+        strong:
+          - cuda-version >={{ cuda_version }},<{{ cuda_version.split(".")[0]|int + 1 }}
     files:
       - etc/conda/activate.d/~cuda-nvcc_activate.sh      # [linux]
       - etc/conda/deactivate.d/~cuda-nvcc_deactivate.sh  # [linux]
     requirements:
       run:
+        # Safeguard for early parsing
+        {% if cross_target_platform is defined %}
```

Comment:
If the conditional logic gets too complex, we could write
```
{% cross_target_platform = cross_target_platform | default("NOT_DEFINED") %}  # Safeguard for early parsing
```
at the top instead.

Reply:
The was needed to get conda-smithy to re-render variants correctly. Tried several things, but this was the only one that worked so far

However since conda-build is having issues, wouldn't be surprised that we need to tweak this somehow